SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracer.h

```diff
@@ -101,4 +101,59 @@ struct SpatialScopedActiveSpanId
 	Trace_EventTracer* EventTracer;
 };
 
+template <typename T>
+FSpatialGDKSpanId SpatialEventTracer::TraceEvent(const char* EventType, const char* Message,
+												 const Trace_SpanIdType* Causes, int32 NumCauses, T&& DataCallback) const
+{
+	if (Causes == nullptr && NumCauses > 0)
```

Comment:
Is this a valid state? e.g. should this be a warning or is this expected behaviour?

Reply:
Invalid state. Warning sounds good.